resources/templates/base.twig

```diff
@@ -4,11 +4,22 @@
     {% block head %}
         <title>{% block title %}{% endblock %}</title>
         <link rel="stylesheet" href="/css/bootstrap.min.css" />
+            <style>
```

Comment:
Sry, its not possible. the `popover` class is only used by javascript generated HTML-markup